harness/determined/common/experimental/trial.py

```diff
@@ -84,7 +84,16 @@ def __init__(self, trial_id: int, session: api.Session):
         self.summary_metrics: Optional[Dict[str, Any]] = None
         self.state: Optional[TrialState] = None
 
-    def logs(
+    def logs(self, *args: Any, **kwargs: Any) -> Iterable[str]:
```

Comment:
something feels a little weird about changing the signature of this method and just passing everything through. passing a bogus arg won't get caught by this method anymore. 

eh, i don't suppose it makes any functional difference at all. and this does save us like 10 lines of code. 🤷